src/libLookup/Lookup.cpp

```diff
@@ -990,6 +994,10 @@ bool Lookup::ProcessGetTxBlockFromSeed(const vector<unsigned char>& message,
                           .GetHeader()
                           .GetBlockNum();
     }
+    else if (lowBlockNum == 0)
+    {
+        lowBlockNum = 1;
```

Comment:
I was trying to understand the reason why `lowBlockNum` is overwritten to 1. Can you clarify a bit?

Reply:
so, as the current convention goes (1,0) ---> gives you the latest blocknum, and giving a request (0,0) --> gives you all the blocks in the blockchain. so keeping that in mind, when lookup recv 0 as low blocknum, it starts to give you the blocknums starting from 0.

Reply:
Okay, got it. Perhaps we can add some comments there together with the code.